### ISSUE TITLE
[Help Wanted] React Image Crop does not render.

### ISSUE THREAD

**Issue by fzs1994** [DominicTobias/react-image-crop#288]
![Screenshot (2)](https://user-images.githubusercontent.com/10684213/62847576-99ce7f80-bcf4-11e9-87ab-f27e7d73f430.png)
I'm able to get image base64 in source of image cropper, but `<ReactCrop>` does not render.

```
let LogoCropper = null;
		if (this.state.logoImage) {
			LogoCropper = (
				<ReactCrop
					src={this.state.logoImage}
					crop={crop}
					onImageLoaded={this.onImageLoaded}
					onComplete={this.onCropComplete}
					onChange={this.onCropChange}
				/>
			);
		}
```

**Comment by DominicTobias** [comment]
Hi did you include the CSS?

**Comment by fzs1994** [comment]
Yes, I did
```
import ReactCrop from 'react-image-crop';
import 'react-image-crop/dist/ReactCrop.css';
```

**Comment by DominicTobias** [comment]
It seems likely that your base64 data is invalid in that case

**Comment by fzs1994** [comment]
I figured out, there was some issue with `.ReactCrop__image` styles in ReactCrop.css
![image](https://user-images.githubusercontent.com/10684213/62924211-43813f80-bdcd-11e9-9d08-b00d85b7efa5.png)

**Overriding `max-height: none` resolves the issue in my case**

**Comment by DominicTobias** [comment]
Interesting, is the crop inside an element that has display: grid by any
chance?

**Comment by Aanchal1103** [comment]
We are facing a similar issue, the height of the ReactCrop Image is 0 and this is happening only on some devices. For me, it worked on Safari but failed on Chrome.
As a workaround, we have passed `min-height` in the `imageStyle` prop.

**Comment by DominicTobias** [comment]
Should be fixed in 8.3.0 (removed max-height property)
